tools/clean_invoice.py

```
@@ -0,0 +1,15 @@
+import sys
+
+from bs4 import BeautifulSoup, Comment
+
+def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)